[2021-08-07 17:02:50,919] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: presentation_pipeline.create_presentation_table_customer 2021-08-07T16:55:00+00:00 [queued]>
[2021-08-07 17:02:50,951] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: presentation_pipeline.create_presentation_table_customer 2021-08-07T16:55:00+00:00 [queued]>
[2021-08-07 17:02:50,953] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-07 17:02:50,954] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-07 17:02:50,955] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-07 17:02:50,972] {taskinstance.py:1107} INFO - Executing <Task(MySqlOperator): create_presentation_table_customer> on 2021-08-07T16:55:00+00:00
[2021-08-07 17:02:50,980] {standard_task_runner.py:52} INFO - Started process 663 to run task
[2021-08-07 17:02:50,994] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'presentation_pipeline', 'create_presentation_table_customer', '2021-08-07T16:55:00+00:00', '--job-id', '25', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/presentation_pipeline.py', '--cfg-path', '/tmp/tmpakmxjilv', '--error-file', '/tmp/tmppqy62k4n']
[2021-08-07 17:02:50,996] {standard_task_runner.py:77} INFO - Job 25: Subtask create_presentation_table_customer
[2021-08-07 17:02:51,067] {logging_mixin.py:104} INFO - Running <TaskInstance: presentation_pipeline.create_presentation_table_customer 2021-08-07T16:55:00+00:00 [running]> on host f68214991f84
[2021-08-07 17:02:51,155] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=presentation_pipeline
AIRFLOW_CTX_TASK_ID=create_presentation_table_customer
AIRFLOW_CTX_EXECUTION_DATE=2021-08-07T16:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-07T16:55:00+00:00
[2021-08-07 17:02:51,156] {mysql.py:70} INFO - Executing: CREATE TABLE IF NOT EXISTS presentation.CUSTOMER (
  `CUSTOMER_KEY` int NOT NULL,
  `NATION_NAME` text NOT NULL,
  `CUSTOMER_ACCTBAL` int NOT NULL,
  `CUSTOMER_MKTSEGMENT` text NOT NULL,
  `INGESTION_TIMESTAMP` datetime DEFAULT CURRENT_TIMESTAMP,
  CONSTRAINT `PK_CUSTOMER` PRIMARY KEY (`CUSTOMER_KEY`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
[2021-08-07 17:02:51,169] {base.py:78} INFO - Using connection to: id: dwh_id. Host: dwh, Port: 3306, Schema: , Login: root, Password: ***, extra: {}
[2021-08-07 17:02:51,182] {dbapi.py:204} INFO - Running statement: CREATE TABLE IF NOT EXISTS presentation.CUSTOMER (
  `CUSTOMER_KEY` int NOT NULL,
  `NATION_NAME` text NOT NULL,
  `CUSTOMER_ACCTBAL` int NOT NULL,
  `CUSTOMER_MKTSEGMENT` text NOT NULL,
  `INGESTION_TIMESTAMP` datetime DEFAULT CURRENT_TIMESTAMP,
  CONSTRAINT `PK_CUSTOMER` PRIMARY KEY (`CUSTOMER_KEY`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;, parameters: None
[2021-08-07 17:02:51,208] {dbapi.py:212} INFO - Rows affected: 0
[2021-08-07 17:02:51,230] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=presentation_pipeline, task_id=create_presentation_table_customer, execution_date=20210807T165500, start_date=20210807T170250, end_date=20210807T170251
[2021-08-07 17:02:51,288] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-07 17:02:51,327] {local_task_job.py:149} INFO - Task exited with return code 0
